rts/Lua/LuaOpenGL.cpp

```diff
@@ -7017,6 +7017,8 @@ int LuaOpenGL::GetMapRendering(lua_State* L)
 
 /***
  * Labels an object for use with debugging tools.
+ * May be unavailable and `nil` if the platform doesn't support the feature.
```

Comment:
What might be `nil`? There is no return value.

Reply:
Oh, I see. This function itself may not exist? It would be nice if the function always existed but did nothing in this case (maybe logs a warning).

Reply:
I thought so too but see #2022.